packages/web-runtime/src/pages/account.vue

```diff
@@ -109,8 +109,8 @@
                 target="_blank"
                 class="oc-flex oc-flex-middle"
               >
+                <oc-icon name="service" size="1" class="oc-mr-xs"></oc-icon>
```

Comment:
Valid `size` values are:
```
'xsmall',
  'small',
  'medium',
  'large',
  'xlarge',
  'xxlarge',
  'xxxlarge'
```

Reply:
Hehe yes. I wanted to check if I want small or standard size. Missed to remove it.